zndraw/server/events.py

```diff
@@ -522,11 +557,82 @@ def scene_update(data: SceneUpdateData):
 
 
 @io.on("selection:run")
-def selection_run(data: SelectionRunData):
+def selection_run(data: dict):
     """Run the selection."""
-    tasks.run_selection.delay(request.url_root, session["token"], data)
+    tasks.run_selection.delay(
+        f"http://127.0.0.1:{current_app.config['PORT']}", session["token"], data
```

Comment:
Should the URL be hardcoded here and below?

Reply:
It refers to localhost, I think this is fine - I had some issues when passing the full `request.url_root` due to SSL or something